doc/sg/c_cpp_coding_style.md

```diff
@@ -414,7 +414,7 @@ __attribute__((section(".crt"))) void _crt(void);
 ### Nonstandard Compiler Builtins
 
 In order to avoid a total reliance on one single compiler, any nonstandard compiler builtins (also known as intrinsics) should be used via a single canonical definition.
-This ensures changes to add compatibilty for other compilers are less invasive, as we already have a function to include a full implementation within.
+This ensures changes to add compatibility for other compilers are less invasive, as we already have a function to include a full implementation within.
```

Comment:
nit: This ensures changes **that** add compatibility for other compilers are less invasive, as we already have a function **that includes a full implementation**.

Reply:
Fixed